corehq/apps/reminders/views.py

```diff
@@ -176,6 +177,25 @@ def keyword_form(self):
             readonly=self.readonly,
         )
 
+    def can_view_data(self, request):
+        if self.readonly:
+            return True
+
+        if self.keyword.upstream_id:
```

Comment:
nit
```suggestion
        if self.keyword.upstream_id and not self.readonly:
```

Reply:
I think this change negatively affects readability. If you were describing what this was doing in english to someone, I don't believe you'd phrase it this way.